### ISSUE TITLE
properties without range

### ISSUE THREAD

**Issue by pennyl67** [pennyl67/LexMeta#4]
There are various properties re-used from other vocabularies without a range (e.g. dcterms:publisher) and others that do (e.g. dcterms:creator). Not sure if this is done on purpose. I think we should be consistent (to the extent possible) for documentation purposes at least.

**Comment by dlindem** [comment]
I agree that we should as many ranges as we can explicitly. dct:creator had ms:Actor as range, I have added the same to dct:publisher. But, is that correct?

**Comment by dlindem** [comment]
At the moment, our "Agent" is not prov:Agent but ms:Actor, see https://lexbib.elex.is/wiki/Item:Q14

**Comment by pennyl67** [comment]
That's fine; we will add owl:equivalentClass between ms:Agent and prov:Agent. Added to my to-do's
